### PR TITLE
Add new links to share tutorials

### PR DESCRIPTION
**Pull request by LissetteCPR**
[Card](https://trello.com/c/40BV2Nox/87-as-a-user-id-like-to-be-able-to-share-tutorials-with-my-social-media-account-so-that-i-dont-have-to-search-for-the-tutorial-myse)

links-tutorials

### CODE REVIEW

File: templates/tutorial.html [hunk 1]
```diff
@@ -11,6 +11,10 @@
       $endif$
       <br/>
       by $author-name$
+      <br/>
+	$if(tags)$
```
Comment (juanpaucar) [inline templates/tutorial.html]: Could you please check the alignment here or something else missing, tag is appearing regardless of the existence of tags in the post
image

File: templates/tutorial.html [hunk 2]
```diff
@@ -44,29 +48,41 @@
...
     </p>
+
     <div class="btn-social-icon">
-       <script language="javascript" type="text/javascript">
-	  var url = window.location.href;
-	  linkTwitter = function(){
-	  location.href = "https://twitter.com/intent/tweet?text=$title$&url=" + url; }
+       <script>
+          var loc = window.location.href;
+          function ChangeHref(target,link){
```
Comment (juanpaucar) [inline templates/tutorial.html]: functions are usually named in camel case, this means that the first char must be in lowercase, in this case it should be `changeHref`

File: app/Main.hs [hunk 3]
```diff
@@ -104,15 +103,17 @@ rules env = do
 
   match "templates/*" (compile templateCompiler)
 
+  tags <- buildTags markdownPattern (fromCapture "tags/*.html")
```
Comment (juanpaucar) [inline app/Main.hs]: do we have a `tags` directory to look html from?

Comment (juanpaucar) [inline app/Main.hs]: Why is the passed Pattern here `"tags/*.html"`?

**Review by juanpaucar — CHANGES_REQUESTED** [review]
Please, take a look at the comments

**Comment by LissetteCPR** [comment]
Please take a look to check the changes @juanpaucar

**Review by juanpaucar — APPROVED** [review]
LGTM :shipit:

**Pull request merged**